Sources/SwiftProtobuf/JSONEncoder.swift

```diff
@@ -128,6 +128,18 @@ internal struct JSONEncoder {
         separator = asciiComma
     }
 
+    /// Begin a new extension field
+    internal mutating func startExtensionField(name: String) {
+        if let s = separator {
+            data.append(s)
+        }
+        data.append(asciiDoubleQuote)
+        data.append(asciiOpenSquareBracket)
+        data.append(contentsOf: name.utf8)
+        append(staticText: "]\":")
```

Comment:
Style wise, we use two `append()`s above for the `"[`, but here we use a single `append(staticText:)`.  Should we be a little more consistent, i.e. - pick one pattern or the other?

Reply:
I changed it so we use the `staticText:` form here for both.